release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -3381,3 +3381,23 @@ link:https://access.redhat.com/solutions/6963680[{product-title} 4.8.44 containe
 ==== Updating
 
 To update an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-8-45"]
+=== RHBA-2022:5167 - {product-title} 4.8.45 bug fix update
+
+Issued: 2022-06-29
+
+{product-title} release 4.8.45 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:5167[RHBA-2022:5167] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:5166[RHBA-2022:5166] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6965038[{product-title} 4.8.45 container image list]
+
+[id="ocp-4-8-45-bug-fixes"]
+==== Bug fixes
+* Previously, `NetworkManager-wait-online.service` timed out preventing a connection to be established with `coreos-installer` before it fetched the Ignition config files. With this update, `NetworkManager-wait-online.service` has more time to load allowing `coreos-installer` to fetch Ignition config files. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1983773#[BZ#1983773*])
```

Comment:
```suggestion
* Previously, `NetworkManager-wait-online.service` timed out preventing a connection to `coreos-installer` before it fetched the Ignition config files. With this update, `NetworkManager-wait-online.service` has more time to load allowing `coreos-installer` to fetch Ignition config files. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1983773#[*BZ#1983773*])
```

Reply:
The first sentence was a little confusing now that I really look at it (sorry I didn't catch earlier)

You are also missing a * to make the BZ link text bold.